index.d.ts

```diff
@@ -1,4 +1,21 @@
 declare namespace activeWin {
+	interface Options {
+		/**
+		Disable Accessibility permission check (macOS)
```

Comment:
The docs text should be in sync with the readme text.

I think each option should also describe what the effect of disabling it is, for example, the window title not being available.